functions/api/branch.js

```diff
@@ -6,8 +6,10 @@ const formatDate = require('../common/dateFormatter')
 const joi = require('@hapi/joi');
 const admin = require('firebase-admin');
 const { isAdminOrSuperAdmin, isSuperAdmin } = require('../middleware/auth');
+const functions = require('firebase-functions')
+const { isAdmin } = require('../middleware/auth');
```

Comment:
there is no "isAdmin". Pls. use "isAdminOrSuperAdmin" instead

Reply:
removed